src/Core/MailTemplates/Mjml/components/footer.mjml

```
@@ -45,8 +45,8 @@
       <p style="margin-top: 5px">
         Always confirm you are on a trusted Bitwarden domain before logging
         in:<br />
-        <a href="https://bitwarden.com/">bitwarden.com</a> |
-        <a href="https://bitwarden.com/help/emails-from-bitwarden/">Learn why we include this</a>
+        <a href="https://bitwarden.com/" style="text-decoration:none;color:#175ddc; font-weight:400">bitwarden.com</a> |
+        <a href="https://bitwarden.com/help/emails-from-bitwarden/" style="text-decoration:none; color:#175ddc; font-weight:400">Learn why we include this</a>
```

This might be a little off-base, I'm just this week looking into MJML at all, so my familiarity is still low but trying to grow it.

🤔  I notice [in the head](https://github.com/bitwarden/server/blob/f82552fba93a9a68f7cf71f2d54f72d6bfce4ad1/src/Core/MailTemplates/Mjml/components/head.mjml#L11), we have a MJML style defined for `.link`, which is defined to render as an inline rule set. It does have a heavier `font-weight` (600 vs 400), but otherwise is the same. I don't notice it used other places (yet). Could it be that we should use that here with the `css-class` attribute, and let that propagate to the templates?

If 400 is the weight we prefer, maybe we also update that definition?